app/components/app-star-rating-display.component.scss

```diff
@@ -0,0 +1,26 @@
+.empty-stars {
+  position: relative;
+  display: inline-block;
+  font-size: 15px;
+  font-family: 'Font Awesome 5 Pro';
+  letter-spacing: 3px;
+}
+
+.empty-stars::before {
+  content: '\f005 \f005 \f005 \f005 \f005';
+  font-weight: 300;
+}
+
+.filled-stars {
+  position: absolute;
+  top: 0;
+  left: 0;
+  white-space: nowrap;
+  overflow: hidden;
+  width: 0;
+}
+
+.filled-stars::before {
+  content: '\f005 \f005 \f005 \f005 \f005';
+  font-weight: 900;
+}
```

Comment:
to be extra safe, let's change these to be BEM. So like `star-rating-display__empty-stars`, etc. That way when this CSS is imported there's no change it could clash with our other css.